src/Generator.php

```diff
@@ -311,7 +313,7 @@ public function getTemplatePath()
      */
     public function render($template, $params = [])
     {
-        $view = new View();
+        $view = new View(Yii::getApp(), new Theme());
```

Comment:
Is `Yii::getApp()` ok, or should the $app instance be somehow injected?

Reply:
Yes, application could be injected in constructor.

Reply:
Just to be sure, using constructor injection like this:
```
public function __construct(\yii\base\Application $app) {
    $this->_app = $app;
}
```

would mean that I have to go through all the `$generator = new XXXGenerator()` in the test cases to add `$this->app` as parameter ? Is there a nicer way to do this ?

Also, I put back the init() method and implemented Initiable, but it never gets called (try putting `die("I'm called")`, nothing will happen). What am I missing here ? 🤔